### PR TITLE
Accessibility page

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
To add the footer link for the help page(isn't live yet) and to add the accessibility page and footer link. I have also updated the cookie links to the correct versions of the pages.

### How to review 
Check content looks good and footer links work

### Checklist

* [x] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by rmccar** [comment]
[Accessibility page in footer](https://trello.com/c/ZB0yBzQg/3310-accessibility-page-in-footer)

File: templates/static/accessibility.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  <ul class="list">
+    <li>{{ _("you cannot modify the line height or spacing of text") }}</li>
+    <li>{{ _("the online system for getting a new census code has not been fully tested so parts of it may not be accessible to all users") }}</li>
+    <li>{{ _("the Web chat tool may not be fully accessible") }}</li>
+    <li>{{ _("some page titles may not accurately reflect the content on the page for audio users") }}</li>
+  </ul>
+
+  <h3>{{ _("What to do if you need guidance") }}</h3>
+
+  <p>{{ _("Our <a href='{url}'>guidance leaflets about the census</a> are available:").format(url = "https://census.gov.uk/help/languages-and-accessibility") }}</p>
```
Comment (JamesGardiner) [inline templates/static/accessibility.html:36]: This url doesn't link to anything

File: templates/static/accessibility.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+  <h2>{{ _("What we’re doing to improve accessibility") }}</h2>
+
+  <p>{{ _("We are continuing to work with various charities and local deaf communities to understand their needs for the census.") }}</p>
+
+  <p>{{ _("We are looking at designs to see if it is possible to have British Sign Language on every page. This is challenging as we have been unable to find another service that does this, but we continue to investigate.") }}</p>
+
+  <p>{{ _("We will act on feedback from the 2019 Rehearsal in October.") }}</p>
```
Comment (JamesGardiner) [inline templates/static/accessibility.html:132]: Is October the correct month?

**Review by JamesGardiner — CHANGES_REQUESTED** [review]

File: templates/static/accessibility.html (hunk 2 above)
Comment (rmccar) [reply templates/static/accessibility.html:132]: Its what is in the content I have been given but Ill double check with Charlotte

File: templates/static/accessibility.html (hunk 1 above)
Comment (rmccar) [reply templates/static/accessibility.html:36]: The help section of the campaign website isn't live yet

**Review by JamesGardiner — APPROVED** [review]

File: templates/static/accessibility.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  <ul class="list">
+    <li>{{ _("<a href='{url}'>Equality Advisory and Support Service (EASS)</a> if you live in England or Wales").format(url = "https://www.equalityadvisoryservice.com/") }}</li>
+    <li>{{ _("<a href='{url}'>Equalities Commission for Northern Ireland (ECNI)</a> if you live in Northern Ireland").format(url = "https://www.equalityni.org/Home") }}</li>
+  </ul>
+
+  <h2>{{ _("Technical information about this questionnaire’s accessibility") }}</h2>
+
+  <p>{{ _("The Office for National Statistics is committed to making its website accessible, in accordance with the Public Sector Bodies (Website and Mobile Applications) (No. 2) Accessibility Regulations 2018.") }}</p>
+
+  <p>{{ _("This service is partially compliant with the <a href='{url}'>Web Content Accessibility Guidelines version 2.1</a> AA standard, due to the non-compliances list").format(url = "https://www.w3.org/TR/WCAG21/") }}</p>
```
Comment (ajmaddaford) [inline templates/static/accessibility.html]: "due to the non-compliances list" -> "due to the non-compliances **listed.**"

File: templates/static/accessibility.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
... _("<a href='{url}' download>DAC Report - September 2019</a>").format(url = "https://00e9e64bac927483db2dfd4487850d3028707017bf2b680f4b-apidata.googleusercontent.com/download/storage/v1/b/census-ci-craftcms/o/assets%2FONS-Census-Survey-2021-DAC-Accessibility-Report-Sept-2019.pdf?qk=AD5uMEv4OOShhFARPG-naoLGLStfFwDhiJI9Ppl2oGD8z5UMt3RTdkZOHQupf4-x6E2MOcwbrnCGFPoWF1WTZp0TdnA3QVU1KYuYt3_hpMKS3zcsd1uQESRvF4qU5cRkq34VMSVMKD4ZcPkvab26rSnAt_8qNykO-1aCuOJxNzf0UxhF19b-Pvpy0B5LnzswSkgQN-OmJS7JNKzQMaBelwaSj44Xkg0WSsK32gPxSpxHhL5Bu8DE2ntc8Dm9ppaI01S3WF8xkrOUmj6dtZAXl2AvCHUc64QOddRJKJDh1CHSSMXH_o7YeccP4JVkHXZx4GpO8an62GZVDsAZaZJfesam7A9II5C8nAGmJyYREzhMjgJtz0zxITFLXOVBdj6g9jTXD8VxfdM00YWH_x7fDVPwNUKcVdCTSXWPAlmX66dyj_BEz23guQqOK5pyrPlVMfZfBlBBTyg_fk-YiFEO_Ridw6v4z4ABPkiJvWSoeM_gUiNXRs1JZbCqMuVPHAL6iwMb9TTj6Tvu6-XOkLRhhqNQqLFvVU26NP042o30K9Ilk_CWP978sRy7isjuxb_EthQr64wj0TE8TKEBfTubJANRmUCBFBgQ7vgO0lrb-ySSFhXoO_o-870HeAQ7Mefoi6CWaTmH3mbZCP3P2IXz4obWSfWjF8MgB64wiYeAyCVB2Jvg2Hfv3mkP2hasvsUlLYmcFaa9NrRr-PFy_HTOyMQdQYNddyNCjoMu6XWL2_RU8jYazgmxGLvEY-KwvAjDxApWyivBT2u2f5_0-76RR-xggzv5ehNL29FowG3YrZEYbxNJjeaA1uuFWDV_vzKSyRQ-W0Qq0eBWxe6ReK2TKAPixAvUUoYvAv4LXuRGojB-v1MW4F86-w_QAXr1f75yj_FSoglrR4cI") }}</p>
```
Comment (ajmaddaford) [inline templates/static/accessibility.html]: The link doesn't work, and this is a horrible url. We should not be publishing a url like this when we have a cdn we could serve it from.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: templates/static/accessibility.html [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+  <h2>{{ _("What we’re doing to improve accessibility") }}</h2>
+
+  <p>{{ _("We are continuing to work with various charities and local deaf communities to understand their needs for the census.") }}</p>
+
+  <p>{{ _("We are looking at designs to see if it is possible to have British Sign Language on every page. This is challenging as we have been unable to find another service that does this, but we continue to investigate.") }}</p>
+
+  <p>{{ _("We will act on feedback from the 2019 Rehearsal in October.") }}</p>
+
+  <p>{{ _("This statement was prepared on 11 September.") }}</p>
```
Comment (MebinAbraham) [inline templates/static/accessibility.html]: Does it not need `It was last updated on 11 September.`?

File: templates/static/accessibility.html [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+  <ul class="list">
+    <li>{{ _("in Braille") }}</li>
+    <li>{{ _("in British Sign Language (BSL)") }}</li>
+    <li>{{ _("as an easy read version (which uses pictures to explain the census)") }}</li>
+    <li>{{ _("as a Welsh audio") }}</li>
+    <li>{{ _("as large print in English or large print in Welsh") }}</li>
+  </ul>
+
+  <h3>{{ _("Reporting accessibility problems") }}</h3>
```
Comment (MebinAbraham) [inline templates/static/accessibility.html:46]: This has `[REMOVE UNTIL 23 SEPTEMBER]` next to it in the doc?

**Review by MebinAbraham — CHANGES_REQUESTED** [review]

File: templates/static/accessibility.html (hunk 6 above)
Comment (rmccar) [reply templates/static/accessibility.html:46]: Yeah but eq wont be going live till after then so it doesn't matter in this case

File: templates/static/accessibility.html (hunk 4 above)
Comment (rmccar) [reply templates/static/accessibility.html]: Yeah I know what I've done there, ill fix that

File: templates/static/accessibility.html (hunk 5 above)
Comment (rmccar) [reply templates/static/accessibility.html]: Ive checked this and its correct

File: templates/static/accessibility.html [hunk 7]
```diff
@@ -60,7 +60,7 @@ <h2>{{ _("Technical information about this questionnaire’s accessibility") }}<
 
   <p>{{ _("The Office for National Statistics is committed to making its website accessible, in accordance with the Public Sector Bodies (Website and Mobile Applications) (No. 2) Accessibility Regulations 2018.") }}</p>
 
-  <p>{{ _("This service is partially compliant with the <a href='{url}'>Web Content Accessibility Guidelines version 2.1</a> AA standard, due to the non-compliances list").format(url = "https://www.w3.org/TR/WCAG21/") }}</p>
+  <p>{{ _("This service is partially compliant with the <a href='{url}'>Web Content Accessibility Guidelines version 2.1</a> AA standard, due to the non-compliances listed").format(url = "https://www.w3.org/TR/WCAG21/") }}</p>
```
Comment (ajmaddaford) [inline templates/static/accessibility.html]: Missing a full stop after "listed".

File: templates/static/accessibility.html [hunk 8]
```diff
@@ -119,11 +117,11 @@ <h3>{{ _("Digital Accessibility Centre testing") }}</h3>
...
+  <p>{{ _("<a href='{url}' download>DAC Report - June 2019</a>").format(url = "https://storage.cloud.google.com/census-ci-craftcms/assets/ONS-Census-Survey-2021-DAC-Accessibility-Report-June-2019.pdf") }}</p>
```
Comment (ajmaddaford) [inline templates/static/accessibility.html:124]: This link is still broken.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: templates/static/accessibility.html [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+
+  <p>{{ _("We are continuing to work with various charities and local deaf communities to understand their needs for the census.") }}</p>
+
+  <p>{{ _("We are looking at designs to see if it is possible to have British Sign Language on every page. This is challenging as we have been unable to find another service that does this, but we continue to investigate.") }}</p>
+
+  <p>{{ _("We will act on feedback from the 2019 Rehearsal in October.") }}</p>
+
+  <p>{{ _("This statement was prepared on 17 September.") }}</p>
+
+  <p>{{ _("This statement was published on 17 September.") }}</p>
```
Comment (MebinAbraham) [inline templates/static/accessibility.html]: This should be part of the same sentence as above? Not a new para according to docs

File: templates/static/accessibility.html (hunk 8 above)
Comment (rmccar) [reply templates/static/accessibility.html:124]: Yeah it will be fixed soon. I just need the file to be renamed on the cms for the link to work

**Comment by MebinAbraham** [comment]
You need to update your translation dependency and regenerate translations. Run `pipenv install --dev`

**Review by ajmaddaford — APPROVED** [review]

**Review by MebinAbraham — APPROVED** [review]
Your fate will be decided by Travis.

**Pull request merged**